### ISSUE TITLE
Error during boot. Component not loaded.

### ISSUE THREAD

**Issue by jaimevisser** [ualex73/monitor_docker#78]
```
Logger: custom_components.monitor_docker
Source: custom_components/monitor_docker/__init__.py:118
Integration: Monitor Docker (documentation, issues)
First occurred: 21:24:00 (1 occurrences)
Last logged: 21:24:00

invalid literal for int() with base 10: '1-ce'
```

When booting HA. Config:

```
monitor_docker:
  - name: Docker
```

**Comment by b10126** [comment]
Seems similar to my error log entry

Dieser Fehler wurde von einer benutzerdefinierten Integration verursacht

Logger: custom_components.monitor_docker
Source: custom_components/monitor_docker/__init__.py:118
Integration: Monitor Docker (documentation, issues)
First occurred: 08:19:17 (1 occurrences)
Last logged: 08:19:17

invalid literal for int() with base 10: '7-qnap3'

**Comment by ualex73** [comment]
Can you try the latest 1.12-beta pre-release or download __init__.py and see if it solves the issue?

**Comment by jaimevisser** [comment]
Updated and it still happens.

**Comment by ualex73** [comment]
@jaimevisser are you using a QNAP?

**Comment by jaimevisser** [comment]
@ualex73 Nope, just a plain old docker on Ubuntu.

**Comment by ualex73** [comment]
@jaimevisser  which Ubuntu version are you running? Also, can you do the following:
* Supply the output of "docker info"
* Enable debug logging in the "logger" section by adding the line:
custom_components.monitor_docker: debug
* Restart Home Assistant, and supply the "home-assistant.log", please leave the docker information in there (thus do not remove too much when you remove system specific information)

**Comment by jaimevisser** [comment]
Docker info:
```Containers: 21
 Running: 21
 Paused: 0
 Stopped: 0
Images: 36
Server Version: 18.03.1-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 773c489c9c1b21a6d78b5c538cd395416ec50f88
runc version: 4fc53a81fb7c994640722ac585fa9ca548971871
init version: 949e6fa
Security Options:
 apparmor
 seccomp
  Profile: default
Kernel Version: 5.4.0-86-generic
Operating System: Ubuntu 20.04.3 LTS
OSType: linux
Architecture: x86_64
CPUs: 12
Total Memory: 62.8GiB
Name: Altair
ID: UWNI:OKWM:KIM7:T4NS:73O3:UEZJ:R5JY:JVD5:3QNE:UKNO:6GEW:G3JD
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Labels:
Experimental: false
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false

WARNING: No swap limit support```

**Comment by jaimevisser** [comment]
Log:
```
2021-11-30 19:26:29 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration monitor_docker which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-30 19:26:29 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration unifiprotect which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-30 19:26:29 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration luxtronik which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-30 19:26:29 WARNING (SyncWorker_3) [homeassistant.loader] We found a custom integration hacs which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-30 19:26:29 WARNING (SyncWorker_4) [homeassistant.loader] We found a custom integration nodered which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-30 19:26:29 WARNING (SyncWorker_5) [homeassistant.loader] We found a custom integration afvalwijzer which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-30 19:26:29 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration tesla_custom which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-11-30 19:26:30 DEBUG (Thread-3) [custom_components.monitor_docker.helpers] [Docker]: Helper version: 1.11
2021-11-30 19:26:30 ERROR (Thread-3) [custom_components.monitor_docker] invalid literal for int() with base 10: '1-ce'
2021-11-30 19:26:30 ERROR (Thread-3) [custom_components.monitor_docker] Retry in 60 seconds
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='2-49-1-UNKNOWN (0x00)') on device 'Mancave light' (Node(node_id=2)) will be skipped: Value Value(value_id='2-49-1-UNKNOWN (0x00)') has unknown data in the following location: metadata.cc_specific.sensorType. A reinterview of node Node(node_id=2) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='6-50-1-value-1114113') on device 'Second bedroom light' (Node(node_id=6)) will be skipped: Value Value(value_id='6-50-1-value-1114113') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=6) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='20-50-1-value-513') on device 'Bathroom Mirror' (Node(node_id=20)) will be skipped: Value Value(value_id='20-50-1-value-513') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=20) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='22-50-0-value-197888') on device 'Washing machine' (Node(node_id=22)) will be skipped: Value Value(value_id='22-50-0-value-197888') has unknown data in the following location: metadata.cc_specific.scale. A reinterview of node Node(node_id=22) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='22-50-0-value-1115137') on device 'Washing machine' (Node(node_id=22)) will be skipped: Value Value(value_id='22-50-0-value-1115137') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=22) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='22-50-0-value-197889') on device 'Washing machine' (Node(node_id=22)) will be skipped: Value Value(value_id='22-50-0-value-197889') has unknown data in the following location: metadata.cc_specific.scale. A reinterview of node Node(node_id=22) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='22-50-0-value-1025') on device 'Washing machine' (Node(node_id=22)) will be skipped: Value Value(value_id='22-50-0-value-1025') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=22) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='39-50-1-value-1') on device 'Frost prevention' (Node(node_id=39)) will be skipped: Value Value(value_id='39-50-1-value-1') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=39) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='39-50-1-value-1114113') on device 'Frost prevention' (Node(node_id=39)) will be skipped: Value Value(value_id='39-50-1-value-1114113') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=39) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='42-50-1-value-1114625') on device 'Ventilation' (Node(node_id=42)) will be skipped: Value Value(value_id='42-50-1-value-1114625') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=42) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='42-50-1-value-1049345') on device 'Ventilation' (Node(node_id=42)) will be skipped: Value Value(value_id='42-50-1-value-1049345') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=42) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='43-50-1-value-590337') on device 'Heating regulator 2nd floor' (Node(node_id=43)) will be skipped: Value Value(value_id='43-50-1-value-590337') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=43) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='43-50-1-value-328193') on device 'Heating regulator 2nd floor' (Node(node_id=43)) will be skipped: Value Value(value_id='43-50-1-value-328193') has unknown data in the following location: metadata.cc_specific.scale. A reinterview of node Node(node_id=43) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='43-50-1-value-1114625') on device 'Heating regulator 2nd floor' (Node(node_id=43)) will be skipped: Value Value(value_id='43-50-1-value-1114625') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=43) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='45-50-1-value-1') on device 'Heatpump' (Node(node_id=45)) will be skipped: Value Value(value_id='45-50-1-value-1') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=45) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='47-50-2-value-590337') on device 'Shed lights' (Node(node_id=47)) will be skipped: Value Value(value_id='47-50-2-value-590337') has unknown data in the following location: metadata.cc_specific.meterType. A reinterview of node Node(node_id=47) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='54-49-0-UNKNOWN (0x81)') on device 'Mancave Sensor' (Node(node_id=54)) will be skipped: Value Value(value_id='54-49-0-UNKNOWN (0x81)') has unknown data in the following location: metadata.cc_specific.sensorType. A reinterview of node Node(node_id=54) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:32 ERROR (MainThread) [homeassistant.components.zwave_js] Discovery for value Value(value_id='59-49-0-UNKNOWN (0x5b)') on device 'Second bedroom sensor' (Node(node_id=59)) will be skipped: Value Value(value_id='59-49-0-UNKNOWN (0x5b)') has unknown data in the following location: metadata.cc_specific.sensorType. A reinterview of node Node(node_id=59) may correct this issue, but if it doesn't, please report this issue as it may be caused by either an upstream issue with the driver or missing support for this data in the library
2021-11-30 19:26:34 WARNING (MainThread) [zigpy_deconz.api] No response to 'Command.aps_data_indication' command with seq id '0x1f'
2021-11-30 19:26:36 WARNING (MainThread) [zigpy_deconz.api] No response to 'Command.aps_data_indication' command with seq id '0x29'
2021-11-30 19:26:54 WARNING (MainThread) [homeassistant.config_entries] Config entry 'HL-3170CDW E71798F8J908512' for brother integration not ready yet: No SNMP response received before timeout; Retrying in background
2021-11-30 19:27:30 DEBUG (Thread-3) [custom_components.monitor_docker.helpers] [Docker]: Helper version: 1.11
2021-11-30 19:27:30 ERROR (Thread-3) [custom_components.monitor_docker] invalid literal for int() with base 10: '1-ce'
2021-11-30 19:27:30 ERROR (Thread-3) [custom_components.monitor_docker] Retry in 60 seconds
2021-11-30 19:27:31 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/buienradar/util.py", line 147, in async_update
    result = parse_data(
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar.py", line 37, in parse_data
    return parse_json_data(content, raincontent,
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar_json.py", line 306, in parse_json_data
    data = __parse_precipfc_data(raincontent, timeframe)
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar_json.py", line 574, in __parse_precipfc_data
    mmu = 10**(float((int(val) - 109)) / 32)
ValueError: invalid literal for int() with base 10: '86,6329598612474'
2021-11-30 19:27:31 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/components/buienradar/util.py", line 147, in async_update
    result = parse_data(
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar.py", line 37, in parse_data
    return parse_json_data(content, raincontent,
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar_json.py", line 306, in parse_json_data
    data = __parse_precipfc_data(raincontent, timeframe)
  File "/usr/local/lib/python3.9/site-packages/buienradar/buienradar_json.py", line 574, in __parse_precipfc_data
    mmu = 10**(float((int(val) - 109)) / 32)
ValueError: invalid literal for int() with base 10: '86,6329598612474'
2021-11-30 19:28:30 DEBUG (Thread-3) [custom_components.monitor_docker.helpers] [Docker]: Helper version: 1.11
2021-11-30 19:28:30 ERROR (Thread-3) [custom_components.monitor_docker] invalid literal for int() with base 10: '1-ce'
2021-11-30 19:28:30 ERROR (Thread-3) [custom_components.monitor_docker] Retry in 60 seconds
2021-11-30 19:29:30 DEBUG (Thread-3) [custom_components.monitor_docker.helpers] [Docker]: Helper version: 1.11
2021-11-30 19:29:30 ERROR (Thread-3) [custom_components.monitor_docker] invalid literal for int() with base 10: '1-ce'
2021-11-30 19:29:30 ERROR (Thread-3) [custom_components.monitor_docker] Retry in 60 seconds
2021-11-30 19:30:10 WARNING (Recorder) [homeassistant.components.sensor.recorder] sensor.attic_landing_light_water_cubic_meters has unknown unit Cubic meters
2021-11-30 19:30:10 WARNING (Recorder) [homeassistant.components.sensor.recorder] sensor.washing_machine_water_consumed_us_gallons has unknown unit US gallons
2021-11-30 19:30:10 WARNING (Recorder) [homeassistant.components.sensor.recorder] sensor.washing_machine_water_consumed_cubic_meters has unknown unit Cubic meters
2021-11-30 19:30:10 WARNING (Recorder) [homeassistant.components.sensor.recorder] sensor.dyson_charger_water_consumed_cubic_meters has unknown unit Cubic meters
2021-11-30 19:30:10 WARNING (Recorder) [homeassistant.components.sensor.recorder] sensor.solar_power has unknown unit W
2021-11-30 19:30:31 DEBUG (Thread-3) [custom_components.monitor_docker.helpers] [Docker]: Helper version: 1.11
2021-11-30 19:30:31 ERROR (Thread-3) [custom_components.monitor_docker] invalid literal for int() with base 10: '1-ce'
2021-11-30 19:30:31 ERROR (Thread-3) [custom_components.monitor_docker] Retry in 60 seconds
2021-11-30 19:31:31 DEBUG (Thread-3) [custom_components.monitor_docker.helpers] [Docker]: Helper version: 1.11
2021-11-30 19:31:31 ERROR (Thread-3) [custom_components.monitor_docker] invalid literal for int() with base 10: '1-ce'
2021-11-30 19:31:31 ERROR (Thread-3) [custom_components.monitor_docker] Retry in 60 seconds
```

**Comment by jaimevisser** [comment]
Ubuntu version:
```
root@Altair:/docker# lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 20.04.3 LTS
Release:        20.04
Codename:       focal
```

**Comment by ualex73** [comment]
Ok, i will try to test with Ubuntu ... Only why are you running an older version of docker? You run 18.03.1-ce, and 20.10.10 is already out.

**Comment by jaimevisser** [comment]
It's the latest version in the official ubuntu repos for 20.04. I could add the docker repo and upgrade.

**Comment by ualex73** [comment]
Ok, i will also test first with the Ubuntu repo version (I myself us the docker.io version directly).

**Comment by jaimevisser** [comment]
Upgrading fixed the problem for me. Entities are showing up.

**Comment by ualex73** [comment]
@jaimevisser  i was searching the Ubuntu 20.04 repository for the docker-ce 18.03.1 release, but it does not seem to exist in there. Did you upgrade your Ubuntu from e.g. 18.04 to 20.04 before?

**Comment by jaimevisser** [comment]
No, it's a clean install (from 2020 though).

Jaime Visser


Op do 2 dec. 2021 om 10:12 schreef Alexander ***@***.***>:

**Comment by ualex73** [comment]
Ok, I found the initial bug :-) version ending with "-ce" was not liked. fixed in 1.12-beta2 I think

**Comment by b10126** [comment]
Good morning, I tested the beta version from yesterday evening and it does work. Environment ist qts 5.x, Homeassistant on docker. There were no errors and the functionality like starting/stopping/monitoring works w/o problems.

**Comment by ualex73** [comment]
Released v1.12 officially, so we can close this one now too ;-)
